files/dashboards/openshift-logging-dashboard.json

```diff
@@ -1678,7 +1769,6 @@
         "value_type": "individual"
       },
       "type": "graph",
-      "span": 2,
```

Comment:
All changesup until here ... seems not really needed please get them back to original

Reply:
yeah I reverted them.